package.json

```diff
@@ -14,7 +14,7 @@
   "private": true,
   "engines": {
     "node": "16.19.0",
-    "pnpm": "7.19.0"
+    "pnpm": "7.21.0"
```

Comment:
This will require a corresponding change in the recipes repo - which update needs to be included in this PR as well.

Reply:
Yes yes, Raised a PR

Reply:
I have approved and merged https://github.com/ferdium/ferdium-recipes/pull/278